pennylane/transforms/hamiltonian_expand.py

```diff
@@ -77,6 +100,25 @@ def hamiltonian_expand(tape):
         raise ValueError(
             "Passed tape must end in `qml.expval(H)`, where H is of type `qml.Hamiltonian`"
         )
-
-    hamiltonian.simplify()
-    return qml.transforms.measurement_grouping(tape, hamiltonian.ops, hamiltonian.coeffs)
+    if group:
+        hamiltonian.simplify()
+        return qml.transforms.measurement_grouping(tape, hamiltonian.ops, hamiltonian.coeffs)
+
+    # create tapes that measure the Pauli-words in the Hamiltonian
+    tapes = []
+    for ob in hamiltonian.ops:
+        with tape.__class__() as new_tape:
+            for op in tape.operations:
+                qml.apply(op)
+            qml.expval(ob)
```

Comment:
I'm curious if you could get away with
```python
for ob in hamiltonian.ops:
    new_tape = tape.copy()
    new_tape.measurements = [MeasurementProcess(return_type=qml.operation.Expectation, obs=ob)]
    tapes.append(new_tape)
```

Reply:
I imagine this is a tiny bit faster?

Reply:
I shall try that, but had issues with setting properties that do not have a setter...but maybe that is another case...

Reply:
Oh, I know why - it needs to be

```python
new_tape._measurements = [MeasurementProcess(return_type=qml.operation.Expectation, obs=ob)]
```

Reply:
Seems to work :)